amt/charts/mps/templates/middleware-func-cm.yaml

```diff
@@ -0,0 +1,86 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mps-middleware-configmap
+data:
+  multiTenancyMiddleware.js: |
+    import express from 'express'
```

Comment:
I don't think you're using `express` anywhere, probably don't need the import

Reply:
Removed